tests/test.c

```diff
@@ -116,6 +116,12 @@ child(int master, int slave)
 	if (ioctl(slave, TIOCSCTTY, NULL) < 0)
 		err(1, "ioctl");
 
+	/* Enable malloc.conf(5) options on OpenBSD which will abort the pick
+	 * process when reading/writing out-of-bounds or accessing already freed
+	 * memory. */
+	if (setenv("MALLOC_OPTIONS", "S", 0) < 0)
```

Comment:
I used the environment variable since the current process is replaced
when calling `execvp`.